Donell/donell_restaurants_challenge.py

```diff
@@ -0,0 +1,138 @@
+
+# TODO: Write code to print the latitude and longitude of Four Barrel Coffee.
+
+print({restaurant["latitude"]})
+print({restaurant["longitude"]})
+
+# TODO: Write code to print the complete address of the Four Barrel Coffee, formatted as a string - it should include the address, city, state and the zip code.
+print(f'Four Barrel Coffee restaraunt is located at {restaurant["address1"]} {restaurant["city"]} {restaurant["state"]} {restaurant["zip_code"]}')
+
+
+# TODO: Write code to print the URL of the website of Four Barrel Coffee.
+print(restaurant["url"])
+
+
+print()
+
+print("Question 2")
+
+# TODO: Choose 3 of your most favourite restaurants in NYC, and create a dictionary for each of them with the following key-value pairs:
+#         1. name : name of the resturant (string)
+#         2. address: address of the restaurant (string)
+#         3. favourite_dish: your favourite thing to order at the restaurant (string)
+
+
+restaurant_1 = {
+    "name": "Papa Johns",
+    "address": "250 East 151st Bronx, Ny 10453",
+    "favourite_dish": "Pizza Pie"
+}
+
+restaurant_2 = {
+    "name": "McDonalds",
+    "address": "260 West 125th NY, NY 10015",
+    "favourite_dish": "Big Mack"
+}
+
+restaurant_3 = {
+    "name": "Taco Bell",
+    "address": "270 West 110th St NY, NY 10013",
+    "favourite_dish": "Tacos"
+}
+
+# TODO: Print each dictionary
+
+print(restaurant_1)
+print(restaurant_2)
+print(restaurant_3)
+
+# The dictionary for each restaurant should look something like this
+
+'''
+restaurant_1  = {
+    "name": "Subway",
+    "address" : "116th & Broadway, NY 10016",
+    "favourite_dish" : "Chicken BLT Sandwich"
+}
+'''
+
+print()
+
+print("Question 3")
+'''
+Imagine that any 1 of your most favourite restaurants stopped serving your favourite dish there.
+Remove the 'favourite_dish' key value pair from that restaurant's dictionary
+'''
+
+# TODO: Remove the 'favourite_dish' key-value pair from one of your 3 restaurants
+
+restaurant_1.pop("favourite_dish")
+
+# TODO: Print the new dictionary. The new dictionary should only contain 'name' and 'address' for that restaurant
+
+print(restaurant_1)
+
+print()
+
+print("Question 4")
+'''
+Imagine that another one of your most favourite restaurants modified its address.
+Update just this value in that restaurant's dictionary
+'''
+
+# TODO: Update the address field of 1 restaurant
+restaurant_2['address'] = "473 Prospect Aveune"
+# TODO: Print the new address of the restaurant by accessing that field of the restaurant's dictionary
+print(restaurant_2["address"])
+
+
+# TODO: Print the updated dictionary.
+print(restaurant_2) 
+
+print()
+
+
+print("Question 5")
+'''
+Printing out all 3 of our restaurants every time is getting annoying. Let's clean up our code!
+'''
+
+# TODO: Put your 3 restaurant dictionaries into a list called `restaurants`
+
+restaurants = [restaurant_1, restaurant_2, restaurant_3]
+
+# TODO: Loop through your list and print out the name and address of each restaurant
+ 
```

Comment:
You for got this part. You need to use a for loop to loop through the list you created on line 135.